### PR TITLE
Feat 355 add related projects

### PR DESCRIPTION
**Pull request by JCoronaWize**
#### What does this PR do?
Adds the related projects Model
Includes select input to add related projects
Creates and Deletes relations between projects when updating projects in projects Page
Adds Related projects section in projects detail's page

#### Where should the reviewer start?
Check the created the new model: RelatedProjects on `schema.prisma`
Run the project a go to a projects **Edit Page** to view the new `related projects `field

#### How should this be manually tested?

1. Go to a projects edit Page.
2. Add projects on the `related projects `input, and save
3. On the **_Project's details_** page check that on **_Related Projects_** section the information is updated.

#### Any background context you want to provide?
This  a featured meant to allow users to see projects that are related to each other, which relation is created by the project members and/or admin users. 
This MVP is a feature that would be added to the projects new version using REMIX

#### What are the relevant tickets?
This is related to #355 

#### Screenshots
Captura de Pantalla 2022-08-11 a la(s) 14 27 31

Captura de Pantalla 2022-08-11 a la(s) 14 28 21

#### Questions
We could use other UI element instead of `Chips` if we wish to add more information.
Would this feature be added to other sections ? eg: `Admins Page`

### CODE REVIEW

**Review by LuisTejedaS — CHANGES_REQUESTED** [review]
just minor changes

File: app/core/components/RelatedProjectsSelect/index.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
+import { Fragment, PropsWithoutRef, useState } from "react"
+import { useQuery } from "blitz"
+import { CircularProgress, TextField, Autocomplete } from "@mui/material"
+import { Field } from "react-final-form"
+// import getSkills from "app/skills/queries/getSkills"
```
[the remark this answers is not in the document]
Comment (JCoronaWize) [reply app/core/components/RelatedProjectsSelect/index.tsx]: Sure thing

**Review by LuisTejedaS — APPROVED** [review]
There is still a commented line, you can either delete it and then merge or merge and then we just delete it

**Comment by JCoronaWize** [comment]
Removed, thank you

**Pull request merged**